pkg/sdk/database_role_test.go

```diff
@@ -15,13 +17,13 @@ func TestDatabaseRoleCreate(t *testing.T) {
 
 	t.Run("validation: nil options", func(t *testing.T) {
 		var opts *createDatabaseRoleOptions = nil
-		assertOptsInvalidJoinedErrors(t, opts, errNilOptions)
+		assertOptsInvalidJoinedErrors(t, opts, ErrNilOptions)
```

Comment:
should errors also be move into the internal package? Especially since jan is going to do some work with that. i guess it work make package imports weird. like errors.ErrNilOptions, also golang doesn't like it when you overwrite the standard package names

Reply:
I will move them (to package `sdkerr` for now). And we will revisit this topic with the errors rework. cc: @sfc-gh-jcieslak